elements/pf-chip/pf-chip-group.ts

```diff
@@ -0,0 +1,249 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import type { PropertyValues } from 'lit';
+import { query } from 'lit/decorators/query.js';
+import { queryAssignedNodes } from 'lit/decorators/query-assigned-nodes.js';
+import { RovingTabindexController } from '@patternfly/pfe-core/controllers/roving-tabindex-controller.js';
+import { PfChip, ChipReadyEvent, ChipRemoveEvent } from './pf-chip.js';
+
+import styles from './pf-chip-group.css';
+/**
+ * A **chip group** is a collection of chips that can be grouped by category and used to represent one or more values assigned to a single attribute. When value of numChips is exceeded, additional chips will be hidden using an overflow chip.
+ *
+ * @slot category-name
+ *      Category name text for chip group category. If this prop is supplied chip group with have a label and category styling applied
+ *
+ * @slot
+ *      Should be <Chip> elements.
+ *
+ * @fires { Event } overflow-chip-click - when close button is clicked
+ */
+@customElement('pf-chip-group')
+export class PfChipGroup extends LitElement {
+  static readonly styles = [styles];
+  static override readonly shadowRootOptions: ShadowRootInit = { ...LitElement.shadowRootOptions, delegatesFocus: true };
+
+  /**
+   * Accessible label for chip group that does not have a category name
+   */
+  @property({ type: String }) label = '';
+
+  /**
+   * Accessible label for close button
+   */
+  @property({ attribute: 'accessible-close-label', type: String }) accessibleCloseLabel = 'Close';
+
+  /**
+   * Flag if chip group can be closed
+   */
+  @property({ type: Boolean }) closeable = false;
+
+  /**
+   * Customizeable "more" template string. Use variable "${remaining}" for overflow chip count.
```

Comment:
```Suggestion
   * Customizable "more" template string. Use variable "${remaining}" for overflow chip count.
```

Reply:
Not a blocker

Also wonder if we should use `{{ x }}` placeholder pattern instead of `${ x }` and describing it as a variable like string literal?

I'd be curious what feels more familiar to users.

```
  /**
   * Customizable "more" template string. Use placeholder "{{remaining}}" for overflow chip count.
   */
  @property({ attribute: 'collapsed-text', type: String }) collapsedText = '{{remaining}} more';
  
  ... 
  
    if (this.collapsedText.match(/\{\{remaining\}\}/)) {
      collapsedText = this.collapsedText.split('{{remaining}}').join(`${this.remaining}`);
    }
```

Reply:
these regexps are suprisingly difficult to get right. see the project `stampino` for example. tldr you need negative lookbehinds, which might crash some safari versions